internal/pkg/build/sources/conveyorPacker_debootstrap.go

```diff
@@ -44,13 +48,35 @@ func (cp *DebootstrapConveyorPacker) Get(b *types.Bundle) (err error) {
 
 	// run debootstrap command
 	cmd := exec.Command(debootstrapPath, `--variant=minbase`, `--exclude=openssl,udev,debconf-i18n,e2fsprogs`, `--include=apt,`+cp.include, `--arch=`+runtime.GOARCH, cp.osversion, cp.b.RootfsPath, cp.mirrorurl)
-	cmd.Stdout = os.Stdout
-	cmd.Stderr = os.Stderr
 
 	sylog.Debugf("\n\tDebootstrap Path: %s\n\tIncludes: apt(default),%s\n\tDetected Arch: %s\n\tOSVersion: %s\n\tMirrorURL: %s\n", debootstrapPath, cp.include, runtime.GOARCH, cp.osversion, cp.mirrorurl)
 
 	// run debootstrap
-	if err = cmd.Run(); err != nil {
+	out, err := cmd.CombinedOutput()
+
+	io.Copy(os.Stdout, bytes.NewReader(out))
+
+	if err != nil {
+		re := regexp.MustCompile(`W: See (/.+/debootstrap/debootstrap.log) for details`)
```

Comment:
Could be simplified to : `filepath.Join(cp.b.RootfsPath, "debootstrap/debootstrap.log")` with a stat check

Reply:
If you say so...

From the code and output that I was working with, I wasn't completely sure that that file would be located in the same place all the time, but I'll go along with this...